src/components/Chat/RemoteVideoContainer.js

```diff
@@ -9,13 +9,12 @@ class RemoteVideoContainer extends React.Component {
     super(props);
   }
 
-  peerVideo(peerLength) {
+  peerVideo(peerLength,riffIds) {
```

Comment:
style: comma separated lists should have a space after the comma
```suggestion
  peerVideo(peerLength, riffIds) {
```